std/process.d

```diff
@@ -3122,7 +3122,7 @@ private auto executeImpl(alias pipeFunc, Cmd, ExtraPipeFuncArgs...)(
     auto r3 = executeShell("exit 123");
     assert(r3.status == 123);
     assert(r3.output.empty);
-    auto r4 = executeShell("echo stderr test, please ignore 1>&2",
+    auto r4 = executeShell("echo -n '\r' 1>&2",
```

Comment:
This won't work on Windows.

Even on POSIX, I don't think this is portable. It is better to use `printf`. (Which still won't work on Windows. See the test above.)

Reply:
I'll try the same trick as above - reopen stderr with the null handle.